src/components/users/TickForm.tsx

```diff
@@ -7,6 +7,7 @@ import { graphqlClient } from '../../js/graphql/Client'
 import { MUTATION_ADD_TICK } from '../../js/graphql/gql/fragments'
 import ComboBox from '../ui/ComboBox'
 import * as Yup from 'yup'
+import { InformationCircleIcon } from '@heroicons/react/20/solid'
```

Comment:
We're deprecating `@heroicons`.  Please use https://phosphoricons.com/

Example: https://github.com/OpenBeta/open-tacos/blob/develop/src/components/media/PhotoUploadButtons.tsx#L1